### PR TITLE
[Style] Remove line length violations from ingest actions

### PR DESCRIPTION
**Pull request by jdconrad**
Removes all line length violations from ingest actions to get rid the need for checkstyle suppression in this package.

Parent issue: https://github.com/elastic/elasticsearch/issues/34884

### CODE REVIEW

**Review by hub-cap — APPROVED** [review]

**Review by dakrone — APPROVED** [review]
LGTM assuming CI is happy

**Comment by jdconrad** [comment]
@hub-cap @dakrone Thanks for the reviews!  Will commit as soon as CI passes.

File: server/src/main/java/org/elasticsearch/action/ingest/GetPipelineRequestBuilder.java [hunk 1]
```diff
@@ -22,7 +22,8 @@
 import org.elasticsearch.action.support.master.MasterNodeReadOperationRequestBuilder;
 import org.elasticsearch.client.ElasticsearchClient;
 
-public class GetPipelineRequestBuilder extends MasterNodeReadOperationRequestBuilder<GetPipelineRequest, GetPipelineResponse, GetPipelineRequestBuilder> {
+public class GetPipelineRequestBuilder
+    extends MasterNodeReadOperationRequestBuilder<GetPipelineRequest, GetPipelineResponse, GetPipelineRequestBuilder> {
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/action/ingest/GetPipelineRequestBuilder.java]: Could you indent this one more time so the class declaration doesn't line up with the class body?

**Review by nik9000 — APPROVED** [review]
LGTM. I left one thing I'd love for you to change but thanks for cleaning these up!

File: server/src/main/java/org/elasticsearch/action/ingest/GetPipelineRequestBuilder.java (hunk 1 above)
Comment (nik9000) [reply server/src/main/java/org/elasticsearch/action/ingest/GetPipelineRequestBuilder.java]: It is my pet complaint to IntelliJ users :smile:

**Pull request merged**